jujugui/static/gui/src/app/components/entity-content/test-entity-content.js

```diff
@@ -156,26 +162,29 @@ describe('EntityContent', function() {
                 getFile={getFile} />
             </div>
             <div className="four-col">
-              <p>{' '}</p>
+              <juju.components.EntityFiles
+                entityModel={mockEntity}
+                pluralize={pluralize} />
```

Comment:
I thought the `instance` trick was only necessary for callbacks like event handlers, because they get bound. Utility methods aren't bound so no need to use that approach.